operator/v1/types_etcd.go

```diff
@@ -28,12 +28,34 @@ type Etcd struct {
 
 type EtcdSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+	// profile allows user to change the etcd tuning profile which configures
+	// the latency parameters for heartbeat interval and leader election timeouts
+	// allowing the cluster to tolerate longer round-trip-times between etcd members.
+	// Valid values are "Default" and "Slower".
+	// The current default is Default.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=Default
+	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
+	// +optional
+	Profile Profile `json:"profile,omitempty"`
```

Comment:
will this field be force set to Slower for Azure? or are you thinking that empty means select a default.  I'm happy with "a default is chosen", but you need specify in the godoc.  Choosing a default is what we "normally" do and it works extremely well.

Reply:
I was thinking having the default be "Default" for all platforms, so they're consistent, but if we needed to start a platform, like Azure, at a different value, we would set its starting value to Slower.